portfolio/src/main/webapp/travels.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
```

Comment:
Same thing here with the extra tags. https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags